app/public/script.js

```diff
@@ -1,12 +1,51 @@
+const e = require("express");
+
 const webSocket = new WebSocket(`ws://localhost:${WEBSOCKET_PORT}/`);
 webSocket.onmessage = (event) => {
   console.log(event.data);
 };
 webSocket.addEventListener("open", () => {
   console.log("We are connected");
+  sendMessage("Hi from client");
 });
-const sendMessage = () => {
+
+const sendMessage = (message) => {
   // Sends message to websocket server
-  webSocket.send("Hi from client");
+  // "Hi from client"
+  webSocket.send(message);
+};
+
+document
+  .getElementById("loginForm")
+  .addEventListener("submit", function (event) {
+    event.preventDefault();
+
+    let username = document.getElementById("username").value;
+    let password = document.getElementById("password").value;
+    loginUser(username, password);
+  });
+
+function loginUser(username, password) {
+  fetch("/login", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      username: username,
+      password: password,
+    }),
+  })
+    .then((response) => {
+      if (response.status === 200) {
+        return response.json();
+      } else if (response.status === 400) {
```

Comment:
I recommend changing this to just an `else`. We might get a different status code, and we might even get a 500s code (meaning the server went wrong). We want to throw an error whenever the code is not exactly 200.